### DISCUSSION TITLE
Challenge 2 - Examining Reasons for Eviction Filing

### DISCUSSION THREAD

**Discussion by KathrynMercer** [datakind/datakit-housing-fall-2024#42] [Show and tell]
**TLDR:** Leading causes of eviction have changed over time. Evictions could potentially be reduced further by providing/subsidizing legal counsel to evictees to mitigate "Nuisance" and "Breach [of contract]" causes for eviction.

**What is it:** Visualizations of most frequent causes of eviction in the San Francisco area from 1997-2024 

**What data was used:**

[San Francisco Eviction Notices](https://data.sfgov.org/Housing-and-Buildings/Eviction-Notices/5cei-gny5/about_data) from 1997-2024

**What tools were used:**

Jupyter notebook w/ Python
Pandas, Numpy, and MatPlotLib/Seaborn libraries

**Limitations of the data:**

This data reflects San Francisco eviction filings only. It may or may not be generalizable to the broader populations of LA/CA (or FL!)

This data is eviction FILINGS, which may or may not reflect actual evictions. 

**What does it show:**

The number of evictions decreased substantially during the COVID-19 pandemic and has not (fully) rebounded to previous levels. I don't know what eviction moratorium/other allowances were made during the pandemic in San Francisco, but there was a big decrease in evictions, so probably they worked. 
The causes of eviction have changed substantially over time. 
Highlights:
- "Breach" of lease and "Nuisance" are consistently in the Top 5 reasons for eviction - every year without exception. Since COVID, "Nuisance" has been the most listed cause for filing. The specifics of what Nuisance & Breach likely vary from case to case, but I wonder if getting potential evictee's access to legal assistance to review their leases and what the landlord is considering breaches of contract or nuisance behavior might keep some evictee's in their homes. 
- "Late Payment" and "Nonpayment" made up approximately 10% of the total causes for filing. In particular, the frequency of "Nonpayment" spiked post-COVID (2021-22) but seems to be trending down again in 2023-24. 
- Prior to the 2000's, the most common cause of eviction filing was owner moving into the rental residence. 
- During the 2000's, there was an increase in "Ellis Act Withdrawals" causing eviction filings. [The Ellis Act](https://en.wikipedia.org/wiki/Ellis_Act) states that landlords can evict if they are removing all their listings from the rental market (ex. due to filing for bankruptcy). This uptick might match up with the economic uncertainties of the time. 

**Next Steps:**

Interactable visualization - I want to throw this in Plotly/similar to make a the visualization interactable - allowing the viewer to look at a pie chart of most common causes per year or other customizations.

Generalizing to other areas - I'd love to do something similar with other areas (LA, Florida, etc.), but I haven't found data to do that yet. EvictionLab seems to only have aggregated data & only up to 2018. If anyone feels like contacting the LA Housing Department to get access to more data, that'd be cool. They seem to require landlords to upload "Notice to Terminate" online, but the data can only be searched by address online, so not really helpful to us as-is.

Demographic data - This data (understandably) doesn't publish who is getting evicted, so I couldn't make any generalizations about demographics, income brackets, etc. 

Mapping -  This data can also be organized by zipcode or block to make a geospatial visualization (though I'm less convinced that's helpful). Unless maybe you were going to try to match reason for filing/number of filings with demographics based on zipcode census data.

![540e28d2-ad73-42b4-9978-19ce468437c3](https://github.com/user-attachments/assets/23667f86-fcf4-4203-a1d1-fe194a74ff09)

![79b650c4-b314-45fa-ae2d-92c8bbe9cfb5](https://github.com/user-attachments/assets/0050911a-27b5-4a04-bbe4-3dc1a4089952)
